README.md

```diff
@@ -1,5 +1,8 @@
 # Silverstripe Security Extensions
 
+__NOTE__: This module not longer commercially supported in Silverstripe CMS 5 and it does not provide CMS5-compatible versions.
```

Comment:
```suggestion
__NOTE__: This module is no longer commercially supported in Silverstripe CMS 5 and it does not provide a CMS5-compatible version.
```

Reply:
Thank you. I'll fix this in each PR.
Done.